From: Andreas Pehrson <apehrson@mozilla.com>
Date: Tue, 23 May 2023 06:14:00 +0000
Subject: Bug 1809672 - Refactor TabCapturerWebrtc creation.
 r=ng,webrtc-reviewers

This restores libwebrtc's DesktopCapturer to their upstream state as far as
the tab capturer is concerned.

Differential Revision: https://phabricator.services.mozilla.com/D174290
Mercurial Revision: https://hg.mozilla.org/mozilla-central/rev/3778b2a0e3f93a2077303b91739cdd2eeb590726
---
 modules/desktop_capture/desktop_capturer.h | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/modules/desktop_capture/desktop_capturer.h b/modules/desktop_capture/desktop_capturer.h
index afbf92aff6..aef23ae610 100644
--- a/modules/desktop_capture/desktop_capturer.h
+++ b/modules/desktop_capture/desktop_capturer.h
@@ -179,7 +179,8 @@ class RTC_EXPORT DesktopCapturer {
   // The return value if `pos` is out of the scope of the source is undefined.
   virtual bool IsOccluded(const DesktopVector& pos);
 
-  // Creates a DesktopCapturer instance which targets to capture windows and screens.
+  // Creates a DesktopCapturer instance which targets to capture windows and
+  // screens.
   static std::unique_ptr<DesktopCapturer> CreateGenericCapturer(
       const DesktopCaptureOptions& options);
 
-- 
2.34.1

